src/components/Dashboard/TodoList.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <div id="todo-list">
+    <add-comment-dialog
```

Comment:
This is a duplicate of what's in FilingHistoryList, which is another good reason to move this dialog to where it's actually used (in the DetailsList component).

Reply:
K, will refactor those 3 components, good catch thanks 👍

Reply:
Moving the dialog from the Todo list is no issue and does simplify things, but it does complicate things with regards to the filing history list. Will revisit this one.